### ISSUE TITLE
Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type

### ISSUE THREAD

**Issue by fantasy-fish** [chufengxiao/Example-based-Colorization-via-Dense-Encoding-pyramids#2]
When I ran the following command, I met the error. I am using python2.7 with anaconda on Ubuntu 19.04.
`python test.py -gray ./test_img/gray/1.jpg -refer ./test_img/refer/1.jpg -output ./test_img/result/1.png`

I0123 00:53:42.286531 21910 layer_factory.hpp:77] Creating layer img_l
I0123 00:53:42.286586 21910 net.cpp:86] Creating Layer img_l
I0123 00:53:42.286603 21910 net.cpp:382] img_l -> img_l
I0123 00:53:42.332396 21910 net.cpp:124] Setting up img_l
I0123 00:53:42.332446 21910 net.cpp:131] Top shape: 1 1 64 64 (4096)
I0123 00:53:42.332456 21910 net.cpp:139] Memory required for data: 16384
I0123 00:53:42.332489 21910 layer_factory.hpp:77] Creating layer ref_ab
I0123 00:53:42.332556 21910 net.cpp:86] Creating Layer ref_ab
I0123 00:53:42.332576 21910 net.cpp:382] ref_ab -> ref_ab
I0123 00:53:42.332674 21910 net.cpp:124] Setting up ref_ab
I0123 00:53:42.332692 21910 net.cpp:131] Top shape: 1 2 64 64 (8192)
I0123 00:53:42.332698 21910 net.cpp:139] Memory required for data: 49152
I0123 00:53:42.332708 21910 layer_factory.hpp:77] Creating layer data_l_meansub
I0123 00:53:42.332741 21910 net.cpp:86] Creating Layer data_l_meansub
I0123 00:53:42.332756 21910 net.cpp:408] data_l_meansub <- img_l
I0123 00:53:42.332784 21910 net.cpp:382] data_l_meansub -> data_l
I0123 00:53:42.332887 21910 layer_factory.hpp:77] Creating layer data_l_meansub
I0123 00:53:42.333084 21910 net.cpp:124] Setting up data_l_meansub
I0123 00:53:42.333104 21910 net.cpp:131] Top shape: 1 1 64 64 (4096)
I0123 00:53:42.333112 21910 net.cpp:139] Memory required for data: 65536
I0123 00:53:42.333156 21910 layer_factory.hpp:77] Creating layer data_ab_rs
I0123 00:53:42.333182 21910 net.cpp:86] Creating Layer data_ab_rs
I0123 00:53:42.333194 21910 net.cpp:408] data_ab_rs <- ref_ab
I0123 00:53:42.333217 21910 net.cpp:382] data_ab_rs -> data_ab_rs
I0123 00:53:43.661303 21910 net.cpp:124] Setting up data_ab_rs
I0123 00:53:43.661350 21910 net.cpp:131] Top shape: 1 2 16 16 (512)
I0123 00:53:43.661356 21910 net.cpp:139] Memory required for data: 67584
I0123 00:53:43.661377 21910 layer_factory.hpp:77] Creating layer refer_ab_313
F0123 00:53:43.661437 21910 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Clip, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxCrossEntropyLoss, SoftmaxWithLoss, Split, Swish, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)

**Comment by fantasy-fish** [comment]
I just googled it and uncommented WITH_PYTHON_LAYER:=1 in Makefile.config. Then it worked.
